lib/logflare/partners/partner.ex

```diff
@@ -0,0 +1,27 @@
+defmodule Logflare.Partners.Partner do
+  use Ecto.Schema
+  import Ecto.Changeset
+  alias Logflare.User
+
+  schema "partners" do
```

Comment:
just a thought... maybe a partner should have a user as the main partner "owner", which would let the user self-serve the user management and access token management through the dashboard 🤔

Reply:
I would avoid that for now, there's a lot of uncertainty with this type of user so the leaner we keep it the better.